lib/src/middlewares/ignoreActionTypes.middleware.ts

```diff
@@ -0,0 +1,25 @@
+import { AnyAction, Dispatch, MiddlewareAPI } from 'redux';
+import {
+  REDUX_PENDING_EFFECTS,
+  REDUX_PENDING_EFFECTS_IGNORED_ACTION_TYPES
+} from '../helpers/const';
+
+export const ignoreActionTypesMiddleware = (ignoredActionTypes: string[]) => ({

```

Comment:
I think this function is actually `getIgnoreMiddleware` that returns `ignoreMiddleware` (arrow function under the hook)